src/Container/index.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import RBContainer, { type ContainerProps as RBContainerProps } from 'react-bootstrap/Container';
+
+import type { ComponentWithAsProp } from '../utils/types/bootstrap';
+
+enum ContainerSizeClass {
+  xs = 'container-mw-xs',
+  sm = 'container-mw-sm',
+  md = 'container-mw-md',
+  lg = 'container-mw-lg',
+  xl = 'container-mw-xl',
+}
+
+export type ContainerSize = keyof typeof ContainerSizeClass;
+
+interface ContainerProps extends RBContainerProps {
+  size?: ContainerSize;
+}
+
+type ContainerType = ComponentWithAsProp<'div', ContainerProps> & { Deprecated?: any };
```

Comment:
Is the `Deprecated` property necessary here? `Container` doesn't have a deprecated sub-component (e.g., `Container.Deprecated`) as `Button.Deprecated` does.

Reply:
No, it's not needed. Thank you for catching this.
Fixed here: 641dd15975555d4035920afe71ba2bb8a18c3245